akka-docs/src/main/paradox/typed/from-classic.md

```diff
@@ -108,16 +108,17 @@ A classic actor is started with the `actorOf` method of the `ActorContext` or `A
 
 Corresponding method in Typed is called `spawn` in the @scala[`akka.actor.typed.scaladsl.ActorContext`]@java[`akka.actor.typed.javadsl.ActorContext`].
 
-There is no `spawn` in the @scala[`akka.actor.typed.scaladsl.ActorSystem`]@java[`akka.actor.typed.javadsl.ActorSystem`]
-for creating top level actors. Instead, the single to level actor is defined by a user guardian `Behavior` that
-is given when starting the `ActorSystem`. Other actors are started as children of that user guardian actor or
+There is no `spawn` method in the @scala[`akka.actor.typed.scaladsl.ActorSystem`]@java[`akka.actor.typed.javadsl.ActorSystem`]
+for creating top level actors. Instead, there is a single top level actor defined by a user guardian `Behavior` that is given
+when starting the `ActorSystem`. Other actors are started as children of that user guardian actor or
 children of other actors in the actor hierarchy. This is explained more in @ref:[ActorSystem](#actorsystem).
 
-`actorOf` takes an `akka.actor.Props` parameter, which is like a factory for creating the actor instance, and it's
+The `actorOf` method takes an `akka.actor.Props` parameter, which is like a factory for creating the actor instance, and it's
 also used when creating a new instance when the actor is restarted. The `Props` may also define additional
 properties such as which dispatcher to use for the actor.
 
-The `spawn` method in Typed creates an actor from a given `Behavior` instead of via a `Props` factory.
+Instead of creating an actor via a `Props` factory in classic, in typed, a `spawn` method creates an actor 
+from a given `Behavior`.
```

Comment:
Since we have a typed props perhaps instead:

"In typed, the `spawn` method creates an actor directly from a given `Behavior` without using a `Props` factory. It does however accept an optional `akka.actor.typed.Props` for specifying Actor metadata."

Reply:
+1 for your alternative phrasing.

Reply:
I've added your suggested change.